README.md

```diff
@@ -78,27 +92,34 @@ enable_disk_support = true
 
 gke_config = {
   node_count   = 3
-  # This machine has 128GB RAM
```

Comment:
Since I got rid of the `n2-highmem-32` above, figured I could use the `n2d-highmem-32` here to show the calculation.